docs/install/index.md

```diff
@@ -176,27 +176,112 @@ Download the [onnxruntime-android](https://mvnrepository.com/artifact/com.micros
+  <tr>
+    <th>Device</th>
+    <th>Language</th>
+    <th>PackageName</th>
+    <th>Installation Instructions</th>
+  </tr>
+  <tr>
+    <td>Windows</td>
+    <td>C, C++, C#</td>
+    <!-- TODO (baijumeswani) - Update to Training link once published -->
+    <td><a href="https://www.nuget.org/packages/Microsoft.ML.OnnxRuntime">Microsoft.ML.OnnxRuntime.Training</a></td>
+    <td>
+      <pre lang="bash">dotnet add package Microsoft.ML.OnnxRuntime.Training</pre>
+    </td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>Python</td>
+    <td><a href="https://pypi.org/project/onnxruntime-training/">onnxruntime-training</a></td>
+    <td>
+      <pre lang="bash">pip install onnxruntime-training</pre>
+    </td>
+  </tr>
+  <tr>
+    <td>Linux</td>
+    <td>C, C++</td>
+    <td><a href="https://github.com/microsoft/onnxruntime/releases">onnxruntime-training-linux*.tgz</a></td>
+    <td>
+      <ul>
+        <li>Download the <code>*.tgz</code> file from <a href="https://github.com/microsoft/onnxruntime/releases">here</a>.</li>
+        <li>Extract it.</li>
+        <li>Move and include the header files in the <code>include</code> directory.</li>
+        <li>Move the <code>libonnxruntime.so</code> dynamic library to a desired path and include it.</li>
+      </ul>
+    </td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>Python</td>
+    <td><a href="https://pypi.org/project/onnxruntime-training/">onnxruntime-training</a></td>
+    <td>
+      <pre lang="bash">pip install onnxruntime-training</pre>
+    </td>
+  </tr>
+  <tr>
+    <td>Android</td>
+    <td>C, C++</td>
+    <td><a href="https://mvnrepository.com/artifact/com.microsoft.onnxruntime/onnxruntime-training-android">onnxruntime-training-android</a></td>
+    <td>
+      <ul>
+        <li>Download the <a href="https://mvnrepository.com/artifact/com.microsoft.onnxruntime/onnxruntime-android">onnxruntime-training-android (full package)</a> AAR hosted at Maven Central.</li>
+        <li>Change the file extension from <code>.aar</code> to <code>.zip</code>, and unzip it.</li>
+        <li>Include the header files from the <code>headers</code> folder.</li>
+        <li>Include the relevant <code>libonnxruntime.so</code> dynamic library from the <code>jni</code> folder in your NDK project.</li>
+      </ul>
+    </td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>Java/Kotlin</td>
+    <td><a href="https://mvnrepository.com/artifact/com.microsoft.onnxruntime/onnxruntime-android">onnxruntime-training-android</a></td>
+    <td>In your Android Studio Project, make the following changes to:
+      <ol>
+        <li>build.gradle (Project):
+          <pre lang="gradle">
+repositories {
+    mavenCentral()
+}
+          </pre>
+        </li>
+        <li>build.gradle (Module):
+          <pre lang="gradle">
+dependencies {
+    implementation 'com.microsoft.onnxruntime:onnxruntime-training-android:latest.release'
+}
+          </pre>
+        </li>
+      </ol>
+    </td>
+  </tr>
+</table>
+
+## Large Model Training
```

Comment:
this and "Training install table for all languages" look like duplicates. Can we keep just 1?